### PR TITLE
Improve vite and Typescript support

### PR DESCRIPTION
**Pull request by anbraten**

### CODE REVIEW

File: packages/app/src/views/Home.vue [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <template>
-  <div id="wrapper" class="mx-auto max-w-lg h-full w-full flex flex-col">
+  <div class="app mx-auto max-w-lg h-full w-full flex flex-col">
```
Comment (julianhahn) [inline packages/app/src/views/Home.vue:2]: why has the home component the app class? It's quite confusing when the view does not implement the style which is given in the class.

File: packages/app/src/views/Home.vue [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
     <header class="h-1/5 flex items-end justify-center text-4xl text-gray-800">
       <Button
+        :to="{ name: 'add-tag' }"
```
Comment (julianhahn) [inline packages/app/src/views/Home.vue:5]: image
the button is currently not working

**Review by julianhahn — CHANGES_REQUESTED** [review]

File: packages/app/src/views/Home.vue (hunk 1 above)
Comment (anbraten) [reply packages/app/src/views/Home.vue:2]: Öhhm. Just wanted to get rid of the id as an id has to be unique in html and `wrapper` is quite generic.

File: packages/app/src/views/Home.vue (hunk 2 above)
Comment (anbraten) [reply packages/app/src/views/Home.vue:5]: Good point missed that at while rebasing.

Comment (anbraten) [reply packages/app/src/views/Home.vue:5]: fixed by 8ee28bb52e0a85b39cf9f7815a2edb741473e1a4

**Review by JanEberlein — APPROVED** [review]

File: packages/app/src/views/Home.vue (hunk 1 above)
Comment (anbraten) [reply packages/app/src/views/Home.vue:2]: resolved by e66b6422db31be3a0e73d259933e7c13f5a4d5a7

**Review by julianhahn — APPROVED** [review]
looks good now

**Pull request merged**